src/Components/Artwork/Metadata.tsx

```diff
@@ -27,13 +30,30 @@ export const Metadata: React.FC<MetadataProps> = ({
   hideArtistName,
   hideSaleInfo,
   isHovered,
+  showHoverDetails,
   showSaveButton,
   contextModule,
+  disableRouterLinking,
   ...rest
 }) => {
+  const [linkTo, setLinkTo] = React.useState(artwork.href)
+
+  React.useEffect(() => {
+    if (!!disableRouterLinking) {
+      setLinkTo(window.location.href)
+    } else {
+      if (linkTo !== artwork.href) {
+        setLinkTo(artwork.href)
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [disableRouterLinking])
```

Comment:
Similar to comment above, this doesn't need to be set in a useEffect. Can use `useRouter` hook (from `src/System/Router`) and get access to SSR-friendly `location.href`